src/test/Makefile

```diff
@@ -390,12 +401,14 @@ memcheck-summary-leaks:
 	grep "in use at exit" */memcheck*.log | grep -v " 0 bytes in 0 blocks" || true
 
 check:
+	@echo "Blacklisted tests: $(BLACKLIST_TESTS)"
 	@./RUNTESTS $(RUNTEST_OPTIONS) $(LOCAL_TESTS)
 	$(MAKE) check-remote-quiet-conditional
 	@echo "No failures."
 
 pycheck:
-	@./RUNTESTS.py $(subst nondebug,release,$(RUNTEST_OPTIONS))
+	@echo "Blacklisted tests: $(BLACKLIST_TESTS)"
+	@./RUNTESTS.py $(subst nondebug,release,$(RUNTEST_OPTIONS)) -- $(LOCAL_TESTS)
```

Comment:
I know there are no remote Python tests, but I think this should use TESTS instead of LOCAL_TESTS.

Reply:
done